rosidl_generator_cpp/include/rosidl_generator_cpp/bounded_vector.hpp

```diff
@@ -454,6 +460,16 @@ class BoundedVector
     _Base::push_back(__x);
   }
 
+  template<typename ... Args>
```

Comment:
Could you add a docstring above, as in the other methods?

Reply:
Fixed